src/cljs/athens/events.cljs

```diff
@@ -1045,26 +1045,42 @@
     [prev-block-uid target-rel]))
 
 
+(defn block-save-block-move-composite-op
+  [source-uid ref-uid relation string]
+  (let [block-save-op             (graph-ops/build-block-save-op @db/dsdb source-uid string)
+        block-move-op             (atomic-graph-ops/make-block-move-op source-uid
+                                                                       {:ref-uid ref-uid
```

Comment:
for safety we probably should use `compat` thing.
@filipesilva am I right?

Reply:
@neotyk you are right, as you often are, I would even say always.